fluentforms/core/src/main/java/com/_4point/aem/fluentforms/api/generatePDF/GeneratePDFService.java

```diff
@@ -0,0 +1,59 @@
+package com._4point.aem.fluentforms.api.generatePDF;
+
+import com._4point.aem.fluentforms.api.Document;
+import com._4point.aem.fluentforms.api.Transformable;
+
+public interface GeneratePDFService {
+
+	CreatePDFResult createPDF2(Document inputDoc, String inputFileExtension, String fileTypeSettings,
+			String pdfSettings, String securitySettings, Document settingsDoc, Document xmpDoc) throws GeneratePDFServiceException;
+
```

Comment:
The Adobe API docs outline all the possible values for pdfSettings and securitySettings.  Can we make these enums instead?  That would provide stronger typing than using Strings.

Reply:
Thanks Rob. I was thinking add enum but was not sure...Added now.